.github/workflows/ci.yaml

```diff
@@ -1,6 +1,12 @@
 name: Node.js CI
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
```

@chapulina how about accepting this suggested change as a middle ground?

It will trigger on all commits pushed to this repository directly as before - which is likely only possible for OSRF employees or if their are maintainers with push access, but will additionally trigger on pull_request events which target master, so that if a contributor pushes a commit to their fork, and then opens a PR to master, the jobs will be triggered. By default, github does not enable CI on forks, I had to go into the project settings on my fork to enable testing.

Without this change, the current ```on: [push]``` will not trigger on any PRs which are opened from forks, for example no jobs were triggered for #191

On this PR, the jobs are only being triggered because I touched this file.
On closed PRs you can see CI was not triggered on all PRs https://github.com/osrf/gzweb/pulls?q=is%3Apr+is%3Aclosed

Reply:
Thank you for the explanation, you're right that CI is not triggered for forks. How about enabling CI on all pushes and PRs, independently of branches? The annoying thing is that PRs coming from within this repository will trigger 2 builds, which is not a huge problem.

The reason for enabling it for more branches is that we often have PRs targeted at branches other than master, such as the release branches.

Reply:
:ok_hand: that's actually what I had in 0ea5dbd9d82d8889a174aa8db17179e45124febc

Reply:
I've reverted so it will now trigger on all pushes and all prs regardless of the branch. Definitely easier to maintain and will cover the release branches too :+1: